app/models/user.rb

```diff
@@ -233,6 +233,11 @@ def self.helper_bots
     where(email: ['contact@loomio.org', 'contact@loom.io'])
   end
 
+  def self.email_interceptors
```

Comment:
How keen are you on this bit? I like what I see in the mailers but I don't think this feature should involve stuff in models and queries.